models/DataObject/ClassDefinition/Data/Multiselect.php

```diff
@@ -277,6 +277,17 @@ public function getDataFromEditmode(mixed $data, DataObject\Concrete $object = n
         return $data;
     }
 
+
+    public function getDiffDataFromEditmode(mixed $data, Concrete $object = null, array $params = []): ?array
+    {
+        $data = $data[0]['data'];
+        if (is_string($data) && strlen($data)) {
```

Comment:
```suggestion
        if (is_string($data) && $data !== '') {
```
$data !== '' is slightly faster than strlen and is in my opion more obvious